gef.py

```diff
@@ -10382,6 +10382,14 @@ def __gef_prompt__(current_prompt):
             site.addsitedir(site_packages_dir)
         except FileNotFoundError:
             pass
+        
+        # when Python virtual enviromrnt is used GDB still loads system Python
+        # so GEF doesn't load site-packages dir from environment
```

Comment:
Explain what you're doing to fix it.

Reply:
I tried to explain it in added comments. Hope it is better now.

Reply:
Some suggestions/typos:

* `when` -> `When`
* `enviromrnt` -> `environment`
* `system Python` -> `system-installed Python`